addons/link_tracker/models/link_tracker.py

```diff
@@ -55,7 +55,7 @@ def _compute_absolute_url(self):
             if url.scheme:
                 tracker.absolute_url = tracker.url
             else:
-                tracker.absolute_url = tracker.get_base_url().join(url).to_url()
+                tracker.absolute_url = urls.url_parse(tracker.get_base_url()).join(url).to_url()
```

Comment:
this should be equivalent and less confusing than the original
```suggestion
                tracker.absolute_url = urls.url_join(tracker.get_base_url(), url)
```